### PR TITLE
Remove weird items

### PR DESCRIPTION
**Pull request by matbesancon**
Odd items in documentation files

### CODE REVIEW

File: docs/src/installation.md [hunk 1]
```diff
@@ -125,7 +125,7 @@ specified by using the `SecondOrderCone` set.
 
 ### Gurobi
 
-￼Requires a working installation of Gurobi with an activated license (free for
+Requires a working installation of Gurobi with an activated license (free for
```
Comment (blegat) [inline docs/src/installation.md]: I fail to see what changed

Comment (matbesancon) [reply docs/src/installation.md]: Check http://www.juliaopt.org/JuMP.jl/dev/installation/#MOSEK-1
There is a weird character

Comment (blegat) [reply docs/src/installation.md]: I don't see it, where is it ?

Comment (mlubin) [reply docs/src/installation.md]: I see it in Firefox, not Chrome

**Pull request merged**